java/configuration/src/main/java/sleeper/configuration/properties/table/TablePropertiesExpiry.java

```diff
@@ -0,0 +1,49 @@
+/*
+ * Copyright 2022-2023 Crown Copyright
+ *
+ * Licensed under the Apache License, Version 2.0 (the "License");
+ * you may not use this file except in compliance with the License.
+ * You may obtain a copy of the License at
+ *
+ *     http://www.apache.org/licenses/LICENSE-2.0
+ *
+ * Unless required by applicable law or agreed to in writing, software
+ * distributed under the License is distributed on an "AS IS" BASIS,
+ * WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ * See the License for the specific language governing permissions and
+ * limitations under the License.
+ */
+
+package sleeper.configuration.properties.table;
+
+import org.slf4j.Logger;
+import org.slf4j.LoggerFactory;
+
+import java.time.Duration;
+import java.time.Instant;
+import java.util.HashMap;
+import java.util.Map;
+import java.util.function.Supplier;
+
+public class TablePropertiesExpiry {
+    private static final Logger LOGGER = LoggerFactory.getLogger(TablePropertiesProvider.class);
```

Comment:
The class in the logger should be TablePropertiesExpiry.

Reply:
I've removed this class entirely now.